code/modules/client/preference_setup/loadout/loadout_xeno/vaurca.dm

```diff
@@ -110,4 +110,22 @@
 	cost = 1
 	whitelisted = list("Vaurca Worker", "Vaurca Warrior")
 	sort_category = "Xenowear - Vaurca"
-	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
\ No newline at end of file
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/augment/klaxan_unathi_translator
```

Comment:
I would suggest making them into a single datum entry with multiple choices, so you can't pick both of them.